pkg/connector/backfill.go

```diff
@@ -95,6 +94,35 @@ func (tc *TwitterClient) FetchMessages(ctx context.Context, params bridgev2.Fetc
 			Forward:  params.Forward,
 		}, nil
 	}
+
+	page := 1
+	if params.Forward && params.AnchorMessage != nil && len(messages) > 0 {
+		anchorTS := params.AnchorMessage.Timestamp
+		minMessageTS := methods.ParseSnowflake(messages[0].ID)
+		if minMessageTS.After(anchorTS) {
+			var moreMessages []*types.Message
+			for {
+				messageResp, err := tc.client.FetchConversationContext(ctx, conversationID, &reqQuery, payload.CONTEXT_FETCH_DM_CONVERSATION_HISTORY)
+				if err != nil {
+					break
+				}
+				inbox = messageResp.ConversationTimeline
+				conv = inbox.GetConversationByID(conversationID)
+				chunk := messageResp.ConversationTimeline.SortedMessages(ctx)[conversationID]
+				chunkSize := len(chunk)
+				log.Debug().Int("page", page).Any("size", chunkSize).Msg("FetchConversationContext chunk")
+				moreMessages = append(moreMessages, chunk...)
```

Comment:
These appends are slightly confusing, have you checked that it's always in the correct order if multiple batches are requested

If the order is wrong, might make sense to just append everything into the main `messages` list and sort them afterwards, rather than creating new lists all the time by prepending